src/libraries/System.Private.CoreLib/src/System/TimeSpan.cs

```diff
@@ -308,6 +308,149 @@ public TimeSpan Duration()
 
         public override int GetHashCode() => _ticks.GetHashCode();
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="TimeSpan"/> structure to a specified number of
+        /// days.
+        /// </summary>
+        /// <param name="days">Number of days.</param>
+        /// <returns>Returns a <see cref="TimeSpan"/> that represents a specified number of days.</returns>
+        /// <exception cref="ArgumentOutOfRangeException">
+        /// The parameters specify a <see cref="TimeSpan"/> value less than <see cref="MinValue"/> or greater than <see cref="MaxValue"/>
+        /// </exception>
+        public static TimeSpan FromDays(int days) => FromDays(days, 0);
```

Comment:
It might be worth optimizing these ones (that take a single arg) directly rather than deferring them down. There's quite a bit of code that can be skipped and they could all use a single shared helper that looks like:
```csharp
public static TimeSpan FromDays(int days) => FromUnit(days, TicksPerDay, MinDays, MaxDays);

[MethodImpl(MethodImplOptions.AggressiveInlining)]
internal static TimeSpan FromUnit(int unit, long ticksPerUnit, long minUnits, long maxUnits)
{
    Debug.Assert(minUnits < 0);
    Debug.Assert(maxUnits > 0);

    if ((unit > maxUnits) || (unit < minUnits))
    {
        ThrowHelper.ThrowArgumentOutOfRange_TimeSpanTooLong();
    }
    return TimeSpan.FromTicks(units * TicksPerUnit);
}
```

We could also just replicate the basically 3 lines of code in each case, it's not a lot. No preference on my end.

Note that I don't think it needs to be done in this PR, but its certainly easier to get in if we did it here.